app/assets/images/payment_icons/v_pay.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-v-pay"><title id="pi-v-pay">V Pay</title><path d="M35.712 23.5H2.288A1.787 1.787 0 01.5 21.731V2.269A1.788 1.788 0 012.289.5H35.71a1.788 1.788 0 011.789 1.769v19.462c0 .973-.797 1.769-1.788 1.769h.001z" fill="#fff" fill-rule="nonzero" stroke="#1c3378" stroke-width="1"/><path d="M35.712 23.5H2.288A1.787 1.787 0 01.5 21.731v-6.439h36.999v6.439c0 .973-.797 1.769-1.788 1.769h.001zM25.16 2.206l-4.844 11.565H17.15l-2.384-9.232c-.144-.567-.27-.775-.709-1.017-.723-.388-1.91-.755-2.954-.984l.07-.332h5.091c.648 0 1.232.431 1.38 1.176l1.261 6.692 3.113-7.868h3.142" fill="#1c3378" fill-rule="nonzero"/><path d="M13.465 19.716c.118.031.276.04.472.04.722 0 1.162-.353 1.162-.968 0-.584-.407-.896-1.077-.896-.264 0-.454.024-.557.048v1.776zm-.957-2.446a8.385 8.385 0 011.477-.109c.707 0 1.223.148 1.561.428.314.257.518.669.518 1.162 0 .497-.157.909-.456 1.192-.385.38-.988.56-1.671.56-.179 0-.346-.01-.472-.032v1.981h-.957V17.27zM19.697 20.238l-.415-1.293c-.103-.319-.188-.678-.268-.983h-.015c-.078.305-.158.673-.251.983l-.407 1.293h1.356zm-1.514.725l-.456 1.489h-.995l1.694-5.252h1.232l1.72 5.252h-1.036l-.48-1.489h-1.679zM23.41 22.452v-2.191L21.739 17.2h1.097l.636 1.346c.181.392.313.686.456 1.046h.014c.134-.336.285-.663.464-1.046l.637-1.346h1.09l-1.758 3.038v2.214h-.965z" fill="#fff" fill-rule="nonzero"/></svg>
```

Comment:
The build CI complained that "SVG file does not have the appropriate <title> value"

I believe you have to rename `pi-v_pay` (with underscore) so it matches the `payment_icons.yml` name.

Reply:
And I reverted to far when updating the SVG, fixing that!

Reply:
All good now!